### PR TITLE
fix(web-analytics): Fix conversion rate graph

### PR DESCRIPTION
**Pull request by robbie-c**
## Problem

See https://posthog.slack.com/archives/C075D3C5HST/p1727893009605639

This was just wrong.

## Changes

It's now fixed

## Does this work well for both Cloud and self-hosted?

Yes

## How did you test this code?

Used the demo data, did maths in my head to compare to vistors and conversions graphs.

### CODE REVIEW

File: frontend/src/queries/nodes/WebOverview/WebOverview.tsx [hunk 1]
```diff
@@ -136,7 +136,7 @@ const formatPercentage = (x: number, options?: { precise?: boolean }): string =>
     } else if (x >= 1000) {
         return humanFriendlyLargeNumber(x) + '%'
     }
-    return (x / 100).toLocaleString(undefined, { style: 'percent', maximumFractionDigits: 0 })
+    return (x / 100).toLocaleString(undefined, { style: 'percent', maximumSignificantDigits: 2 })
```
Comment (robbie-c) [inline frontend/src/queries/nodes/WebOverview/WebOverview.tsx:139]: drive-by, but makes smaller conv rates look better, 1.4% rather than 1%

**Review by pauldambra — APPROVED** [review]
low context stamp earned entirely for

> ## How did you test this code?

> did maths in my head
